browser/util.js

```diff
@@ -15,56 +15,23 @@ import * as config from './config.js';
  *     frameworks have loaded.
  */
 export function whenFrameworksReady(callback) {
-  debug('whenFrameworksReady');
-  var done = function() {
-    debug('whenFrameworksReady done');
+  // webcomponents-loader.js will eventually fire WebComponentsReady.
+  // If WebComponents is not defined and we cannot find the script,
+  // invoke the callback.
+  if (!window.WebComponents &&
+    !document.querySelector('script[src*="webcomponents-loader.js"]')) {
```

Comment:
What about other files like webcomponents-es5-loader.js, webcomponents-es6-loader.js (proposed renaming of webcomponents-loader.js), webcomponents-lite.js, or webcomponents.js?